src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,665 @@
+
+        private unsafe T[] FetchData<T>(IntPtr data, int size)
+        {
+            var result = new T[size];
+
+            GCHandle handle = GCHandle.Alloc(result, GCHandleType.Pinned);
+            IntPtr target = handle.AddrOfPinnedObject();
+
+            Int64 sizeInBytes = size * Marshal.SizeOf((typeof(T)));
+            Buffer.MemoryCopy(data.ToPointer(), target.ToPointer(), sizeInBytes, sizeInBytes);
+            handle.Free();
+            return result;
+        }
+
+        private Delegate MakeGetter(IRow input, int iinfo)
+        {
+            var info = _bindings.Infos[iinfo];
+            var outInfo = _bindings.OutputCols[iinfo];
+            var tfType = _bindings.OutputTFTypes[iinfo];
+
+            if (tfType == TFDataType.Float)
+            {
+                return Utils.MarshalInvoke(MakeGetter<float>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Double)
+            {
+                return Utils.MarshalInvoke(MakeGetter<double>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Int32)
+            {
+                return Utils.MarshalInvoke(MakeGetter<int>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Int64)
+            {
+                return Utils.MarshalInvoke(MakeGetter<Int64>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Bool)
+            {
+                return Utils.MarshalInvoke(MakeGetter<bool>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.String)
+            {
+                return Utils.MarshalInvoke(MakeGetter<string>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            throw Host.ExceptNotSupp("Tensorflow type not supported");
+        }
+
+        private ValueGetter<VBuffer<T>> MakeGetter<T>(IRow input, int iinfo)
+        {
+            Host.AssertValue(input);
+            // Need to check type here.
+            //Host.Assert(Infos[iinfo].TypeSrc.IsText);
+
+            var info = _bindings.Infos[iinfo];
+            var tfInfo = _bindings.TfColInfo[iinfo];
+            var srcTensorGetters = GetTensorValueGetters(input, iinfo);
+
+            return (ref VBuffer<T> dst) =>
+                {
+                    var runner = _session.GetRunner();
+                    for (int i = 0; i < info.SrcIndices.Length; i++)
+                    {
+                        var inputName = tfInfo.InputColNames[i];
+                        var type = info.SrcTypes[i];
+                        runner.AddInput(inputName, srcTensorGetters[i].GetTensor());
+                    }
+
+                    var tensors = runner.Fetch(_bindings.OutputColNames[iinfo]).Run();
+
+                    Contracts.Assert(tensors.Length > 0);
+
+                    var output = FetchData<T>(tensors[0].Data, _bindings.OutputCols[iinfo].VectorSize);
```

Comment:
>FetchData [](start = 33, length = 9)

FetchData should use the values array of dst instead of creating a new one (you can check to see the common pattern in other transforms: if the length of dst.Values is not sufficient, we create a new array.)